plugins/linkattacher/include/GazeboYarpPlugins/linkattacherserverimpl.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (C) 2013-2015 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#ifndef YARPGAZEBO_LINKATTACHERSERVERIMPL
+#define YARPGAZEBO_LINKATTACHERSERVERIMPL
+
+#include "gazebo/gazebo.hh"
+#include <gazebo/physics/Model.hh>
+#include <gazebo/physics/World.hh>
+#include <gazebo/physics/Joint.hh>
+#include <gazebo/physics/Link.hh>
+#include <gazebo/physics/PhysicsEngine.hh>
+#include <gazebo/common/Events.hh>
+
+#include <yarp/os/Network.h>
+#include <yarp/os/RpcServer.h>
+#include <yarp/os/Property.h>
+#include <yarp/os/Log.h>
+#include <yarp/os/LogStream.h>
+#include <GazeboYarpPlugins/common.h>
+#include <GazeboYarpPlugins/ConfHelpers.hh>
+
+#include <LinkAttacherServer.h>
+
+const std::string LogPrefix = "LinkAttacher: ";
+
+class LinkAttacherServerImpl: public GazeboYarpPlugins::LinkAttacherServer
+{
+private:
+  gazebo::physics::WorldPtr _world;
+  gazebo::physics::ModelPtr _model;
+
+public:
+  LinkAttacherServerImpl();
+  ~LinkAttacherServerImpl();
+
+  /**
+  * Attach any link of the models spawned in gazebo to a link of the robot using a fixed joint.
+  * @param model_name name that identifies model in gazebo (that are already spawned in gazebo)
+  * @param model_link_name name of a the link in the model you want to attach to the robot
+  * @param robot_name name of the robot
+  * @param robot_link_name name of the robot link to which you want to attached the model link
+  * @return true if success, false otherwise
+  */
+  virtual bool attachUnscoped(const std::string& model_name, const std::string& model_link_name, const std::string& robot_name, const std::string& robot_link_name);
```

Comment:
Remove `virtual` and add `override` at the end of the function declaration. This applies to all other methods inherited from `LinkAttacherServer` class

Reply:
I think this are automatically generated classes.

Reply:
They actually are. @Yeshashvitvs ignore comments on files inside the autogenerated folder

Reply:
@Yeshasvitvs A clarification. Is this file (`plugins/linkattacher/include/GazeboYarpPlugins/linkattacherserverimpl.h`) autogenerated? If not my comment applies.

Reply:
@diegoferigo It is the header implementation of `LinkAttacherServer.h` file which is auto generated from the thrift file. So, I prefer to keep it the same as it to be consistent.